llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -810,10 +815,45 @@ define void @foo(i8 %val, ptr %ptr) {
   sandboxir::StoreInst *NewSt =
       sandboxir::StoreInst::create(Val, Ptr, Align(8),
                                    /*InsertBefore=*/Ret, Ctx);
+  // Check if create was volatile
```

Comment:
To keep things consistent we should either remove this comment, or add comments to the rest of the checks. Let's remove it for now. We can add the comments later in a separate PR if needed.

Reply:
I'll remove all instance of it.

Reply:
fixed.